### DISCUSSION TITLE
Revolt chat in a entreprise

### DISCUSSION THREAD

**Discussion by chusterb** [stoatchat/discussions#191] [Q&A]
hi,

in a environnement professionnel, do u think he is possible to block member to send mp in a other member on the same server?

**Comment by insertish** [comment]
I'm not entirely sure what you mean.

**Reply by Rexogamer** [reply]
I think they're referring to someone being able to block a user from messaging another user? I fail to see why that would be needed, though - if a user is receiving harassment or such from another user, surely they'd just block the harasser themselves?
